scripts/update-apk.sh

```diff
@@ -11,12 +11,28 @@ if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/open-
     exit 0
 fi
 
+mkdir $HOME/daily/
+cp -R /home/travis/build/fossasia/open-event-orga-app/app/build/outputs/apk/app-debug.apk $HOME/daily/
```

Comment:
Why this process of creating a directory. Copying apk to it. Then copying to the branch and deleting the directory ? Can't you copy directly to the branch ?